### PR TITLE
Feature/react icons

### PR DESCRIPTION
**Pull request by hvandermerwe**
added the react-icon package to package.json

### CODE REVIEW

File: chatterbox/src/Main/MainArea/Settings/user-settings-component.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+                    <Pencil
+                        onClick={() => this.props.editUserEmailRequest(this.props.user)} 
+                        className="edit"
+                    />
+                </div>
+
+                <Button 
+                    onClick={() => this.props.saveChangesRequest(this.props.user)} 
+                    label="DONE"
+                    style={{
```
Comment (WesleyBuck) [inline chatterbox/src/Main/MainArea/Settings/user-settings-component.js]: added the style to .css

**Review by WesleyBuck — DISMISSED** [review]

File: chatterbox/src/Main/MainArea/Settings/user-settings-component.js (hunk 1 above)
Comment (hvandermerwe) [reply chatterbox/src/Main/MainArea/Settings/user-settings-component.js]: i added a SubmitChanges object which contains the css for the <Button /> i could not connect  it to the external .css file

File: chatterbox/src/Auth/Register/index.js [hunk 2]
```diff
@@ -40,15 +40,15 @@ class Register extends Component {
         // alert("stage "+curr)
         let canNext = this.state.canNext;
         if (canNext) {
-            if (this.state.stage == 1) {
+            if (this.state.stage === 1) {
```
Comment (KimberleighBeyl) [inline chatterbox/src/Auth/Register/index.js:43]: Why setstate

Comment (hvandermerwe) [reply chatterbox/src/Auth/Register/index.js:43]: I don't know. This was not a component that I was working on

Comment (KimberleighBeyl) [reply chatterbox/src/Auth/Register/index.js:43]: Ok can you maybe talk to the person that did this?

**Review by KimberleighBeyl — CHANGES_REQUESTED** [review]

File: chatterbox/src/Auth/Register/index.js (hunk 2 above)
Comment (hvandermerwe) [reply chatterbox/src/Auth/Register/index.js:43]: yes I will do that

**Review by KimberleighBeyl — APPROVED** [review]
On condition!!!

**Pull request merged**